hack/debug-network.sh

```diff
@@ -0,0 +1,729 @@
+#!/bin/bash
```

Comment:
if this is standalone and shipped to customers as part of an image, it should be included in an image in `images/` and not in `hack`